src/main/generated/resources/assets/minestuck/lang/en_us.json

```diff
@@ -991,6 +992,7 @@
   "entity.minestuck.turtle.sound": "...",
   "entity.minestuck.turtle.sound.2": "..!",
   "entity.minestuck.vitality_gel": "Vitality Gel",
+  "filled_map.tagkey[minecraft:worldgen/configured_structure_feature / minestuck:scanner_located]": "Frog Temple",
```

Comment:
This is not a good translation key. Since `MSLootEvents.createFrogTempleMap()` isn't trying to be a general function, how about just hardcoding the translation akin to vanilla?